css/login.css

```diff
@@ -0,0 +1,32 @@
+.grid-layout {
+    display: grid;
+    height: 100vh;
+    grid-template-areas: "header header header"
+                        "... ... ..."
+                        "... login ..."
+                        "... ... ..."
```

Comment:
I think you forgot to add a semicolon to the grid template areas. Everything else looks good.

Reply:
added the semicolon